src/components/Datepicker/Datepicker.scss

```diff
@@ -0,0 +1,20 @@
+@import "react-datepicker/dist/react-datepicker.min.css";
+
+$b: "react-datepicker";
```

Comment:
Waarvoor staat deze `$b`? Ik ben persoonlijk geen fan van variabelen die niet descriptive zijn 🤷

Reply:
Kan ik in volgen, `${vendor-c}` als de c in `${c}` voor class(Name) staat.

Reply:
For the record; `$b(ase)`